airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+import json
```

Comment:
Name of this file should be `test_views.py`. Some apps, like `manage` has so many views that `test_views.py` became so large it needed to be exploded into smaller files. That's why things like `manage/tests/test_views.py` became things like `manage/tests/views/test_something.py`.